src/NuGet.Services.SearchService/App_Start/WebApiConfig.cs

```diff
@@ -174,10 +178,26 @@ private static ApplicationInsightsConfiguration InitializeApplicationInsights(Re
             var heartbeatIntervalSeconds = configuration.Root.GetValue("ApplicationInsights_HeartbeatIntervalSeconds", 60);
 
             var applicationInsightsConfiguration = ApplicationInsights.Initialize(
-                    instrumentationKey,
-                    TimeSpan.FromSeconds(heartbeatIntervalSeconds));
+                instrumentationKey,
+                TimeSpan.FromSeconds(heartbeatIntervalSeconds));
 
             applicationInsightsConfiguration.TelemetryConfiguration.TelemetryInitializers.Add(new AzureWebAppTelemetryInitializer());
+            applicationInsightsConfiguration.TelemetryConfiguration.TelemetryInitializers.Add(new KnownOperationNameEnricher(new[]
+            {
+                GetOperationName<SearchController>(HttpMethod.Get, nameof(SearchController.V3SearchAsync)),
```

Comment:
Should we add V2 search? Autocomplete?

Reply:
I was just adding the endpoints we have SLI/SLO for.

Reply:
There is no harm and may be useful. I'll add em.